<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="2" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-02-07T14:30:49 GMT-05:00" name="Default suite" finished-at="2024-02-07T14:31:58 GMT-05:00" duration-ms="69135">
    <groups>
      <group name="ErrorHandling">
        <method signature="ErrorValidation.productValidationTest(java.util.HashMap)[pri:0, instance:selenium.ErrorValidation@7530ad9c]" name="productValidationTest" class="selenium.ErrorValidation"/>
      </group> <!-- ErrorHandling -->
      <group name="purchase">
        <method signature="ErrorValidation.productValidationTest(java.util.HashMap)[pri:0, instance:selenium.ErrorValidation@7530ad9c]" name="productValidationTest" class="selenium.ErrorValidation"/>
      </group> <!-- purchase -->
    </groups>
    <test started-at="2024-02-07T14:30:49 GMT-05:00" name="Default test" finished-at="2024-02-07T14:31:58 GMT-05:00" duration-ms="69135">
      <class name="selenium.ErrorValidation">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:selenium.ErrorValidation@7530ad9c]" started-at="2024-02-07T14:30:49 GMT-05:00" name="launchApplication" finished-at="2024-02-07T14:31:22 GMT-05:00" duration-ms="32778" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="productValidationTest(java.util.HashMap)[pri:0, instance:selenium.ErrorValidation@7530ad9c]" started-at="2024-02-07T14:31:22 GMT-05:00" name="productValidationTest" data-provider="dataP" finished-at="2024-02-07T14:31:26 GMT-05:00" duration-ms="3405" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=12345678A@a, product=ZARA COAT 3, email=michaelscott@gmail.com}]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.ElementClickInterceptedException">
            <message>
              <![CDATA[Element <button class="btn btn-custom"> is not clickable at point (1610,27) because another element <div class="ngx-spinner-overlay ng-tns-c31-1 ng-trigger ng-trigger-fadeIn ng-star-inserted"> obscures it
Build info: version: '4.13.0', revision: 'ba948ece5b*'
System info: os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.15.7', java.version: '20.0.2'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Command: [68668c50-bad4-4cc9-ac02-b2628ceea0e9, clickElement {id=81da9f88-0619-44ea-b906-68473ad507dc}]
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 122.0.1, moz:accessibilityChecks: false, moz:buildID: 20240205133611, moz:debuggerAddress: 127.0.0.1:34396, moz:geckodriverVersion: 0.34.0, moz:headless: false, moz:platformVersion: 19.6.0, moz:processID: 16608, moz:profile: /var/folders/py/zw2dl9f93zz..., moz:shutdownTimeout: 60000, moz:webdriverClick: true, moz:windowless: false, pageLoadStrategy: normal, platformName: mac, proxy: Proxy(), se:cdp: ws://127.0.0.1:34396/devtoo..., se:cdpVersion: 85.0, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Element: [[FirefoxDriver: firefox on mac (68668c50-bad4-4cc9-ac02-b2628ceea0e9)] -> xpath: //button[@routerlink='/dashboard/cart']]
Session ID: 68668c50-bad4-4cc9-ac02-b2628ceea0e9]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.ElementClickInterceptedException: Element <button class="btn btn-custom"> is not clickable at point (1610,27) because another element <div class="ngx-spinner-overlay ng-tns-c31-1 ng-trigger ng-trigger-fadeIn ng-star-inserted"> obscures it
Build info: version: '4.13.0', revision: 'ba948ece5b*'
System info: os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.15.7', java.version: '20.0.2'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Command: [68668c50-bad4-4cc9-ac02-b2628ceea0e9, clickElement {id=81da9f88-0619-44ea-b906-68473ad507dc}]
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 122.0.1, moz:accessibilityChecks: false, moz:buildID: 20240205133611, moz:debuggerAddress: 127.0.0.1:34396, moz:geckodriverVersion: 0.34.0, moz:headless: false, moz:platformVersion: 19.6.0, moz:processID: 16608, moz:profile: /var/folders/py/zw2dl9f93zz..., moz:shutdownTimeout: 60000, moz:webdriverClick: true, moz:windowless: false, pageLoadStrategy: normal, platformName: mac, proxy: Proxy(), se:cdp: ws://127.0.0.1:34396/devtoo..., se:cdpVersion: 85.0, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Element: [[FirefoxDriver: firefox on mac (68668c50-bad4-4cc9-ac02-b2628ceea0e9)] -> xpath: //button[@routerlink='/dashboard/cart']]
Session ID: 68668c50-bad4-4cc9-ac02-b2628ceea0e9
at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:67)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:484)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:52)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:191)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:196)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:171)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:602)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:224)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:77)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
at jdk.proxy2/jdk.proxy2.$Proxy18.click(Unknown Source)
at ecommerce.pageObjects.Utillities.clickMethod(Utillities.java:48)
at ecommerce.pageObjects.ProductsPage.cart(ProductsPage.java:55)
at selenium.ErrorValidation.productValidationTest(ErrorValidation.java:46)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.ElementClickInterceptedException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- productValidationTest -->
        <test-method is-config="true" signature="closeApplication()[pri:0, instance:selenium.ErrorValidation@7530ad9c]" started-at="2024-02-07T14:31:26 GMT-05:00" name="closeApplication" finished-at="2024-02-07T14:31:28 GMT-05:00" duration-ms="2017" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeApplication -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:selenium.ErrorValidation@7530ad9c]" started-at="2024-02-07T14:31:28 GMT-05:00" name="launchApplication" finished-at="2024-02-07T14:31:39 GMT-05:00" duration-ms="11682" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="productValidationTest(java.util.HashMap)[pri:0, instance:selenium.ErrorValidation@7530ad9c]" started-at="2024-02-07T14:31:39 GMT-05:00" name="productValidationTest" data-provider="dataP" finished-at="2024-02-07T14:31:45 GMT-05:00" duration-ms="5515" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=qwertyuiop@1A, product=IPHONE 13 PRO, email=aparnasabb@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- productValidationTest -->
        <test-method is-config="true" signature="closeApplication()[pri:0, instance:selenium.ErrorValidation@7530ad9c]" started-at="2024-02-07T14:31:45 GMT-05:00" name="closeApplication" finished-at="2024-02-07T14:31:46 GMT-05:00" duration-ms="1204" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeApplication -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:selenium.ErrorValidation@7530ad9c]" started-at="2024-02-07T14:31:46 GMT-05:00" name="launchApplication" finished-at="2024-02-07T14:31:55 GMT-05:00" duration-ms="8471" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="loginValidationTest()[pri:0, instance:selenium.ErrorValidation@7530ad9c]" started-at="2024-02-07T14:31:55 GMT-05:00" name="loginValidationTest" finished-at="2024-02-07T14:31:56 GMT-05:00" duration-ms="1232" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginValidationTest -->
        <test-method is-config="true" signature="closeApplication()[pri:0, instance:selenium.ErrorValidation@7530ad9c]" started-at="2024-02-07T14:31:56 GMT-05:00" name="closeApplication" finished-at="2024-02-07T14:31:58 GMT-05:00" duration-ms="2150" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeApplication -->
        <test-method signature="orderValidation()[pri:0, instance:selenium.ErrorValidation@7530ad9c]" started-at="2024-02-07T14:31:58 GMT-05:00" name="orderValidation" finished-at="2024-02-07T14:31:58 GMT-05:00" depends-on-methods="selenium.ErrorValidation.productValidationTest" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method selenium.ErrorValidation.orderValidation() on instance selenium.ErrorValidation@7530ad9c depends on not successfully finished methods 
[selenium.ErrorValidation.productValidationTest() on instance selenium.ErrorValidation@7530ad9c]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method selenium.ErrorValidation.orderValidation() on instance selenium.ErrorValidation@7530ad9c depends on not successfully finished methods 
[selenium.ErrorValidation.productValidationTest() on instance selenium.ErrorValidation@7530ad9c]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:156)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- orderValidation -->
      </class> <!-- selenium.ErrorValidation -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
